simple_scheduler/jobs/ecs_job.py

```diff
@@ -0,0 +1,122 @@
+"""A sample job that prints string."""
+
+import time
+import logging
+from ndscheduler import job
+
+logger = logging.getLogger(__name__)
+
+try:
+    import boto3
+
+    client = boto3.client('ecs')
+except ImportError:
```

Comment:
IMHO, don't catch error here. Let the error bubble up. If boto is not imported, let it fail hard.